lib/interface_quda.cpp

```diff
@@ -5203,6 +5278,150 @@ void plaqQuda(double plaq[3])
   profilePlaq.TPSTOP(QUDA_PROFILE_TOTAL);
 }
 
+/*
+ * Computes the total, spatial and temporal plaquette averages after loading a gauge field.
+ */
+void plaqLoadGaugeQuda(double plaq[3], void *gauge, QudaGaugeParam *param)
```

Comment:
Can you explain what the point of this function is?  Since we already have it's constituents in the form of `loadGaugeQuda` and `plaqQuda`.  Am I missing something?

Reply:
Cleaned up in https://github.com/lattice/quda/pull/1308/commits/e24bc686fab8a6cf6eaa4a70bb12ce4fdf479cfa , thanks for the offline iterations!